utils/tooling/generateCerts.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# Copyright Strimzi authors.
+# License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+# generate 2 pairs of certificates and private keys for development purposes. Certificates valid for 10 years. Any valid certificate/key pair called `strimzi-ui-server` or `strimzi-ui-mock-admin` could be used for development, in present in the `generated/dev_certs/` directory
```

Comment:
I'm not sure we want 10 years - only because chrome will moan. If it's not too slow, might be worth regenerating on startup?

Reply:
Interesting - I didn't have my chrome complain at me (above/beyond the standard self signed cert). As mentioned above, I wanted to keep openssl optional, hence no hard requirement on it/regeneration. Would a year be more sensible do you think?